### PR TITLE
Implement grants page

### PR DESCRIPTION
**Pull request by melijimenez11**
There are still some details to work through that @mukhtyar, and I will be going over later today

### CODE REVIEW

**Review by elehmer — COMMENTED** [review]
For some reason when running the dev server on Ubuntu/Firefox and clicking on the Grants links on new landing page the page doesn't scroll back up to the top of the Grants page.

**Comment by elehmer** [comment]
> For some reason when running the dev server on Ubuntu/Firefox and clicking on the Grants links on new landing page the page doesn't scroll back up to the top of the Grants page.

It scrolls back up just fine on Ubuntu/Chromium...

**Comment by melijimenez11** [comment]
> For some reason when running the dev server on Ubuntu/Firefox and clicking on the Grants links on new landing page the page doesn't scroll back up to the top of the Grants page.

I'm gonna take a look at the issue. Thanks for pointing it out

**Comment by mukhtyar** [comment]
@elehmer @melijimenez11 I think we have encountered this issue with scroll position before in Firefox. It works fine in Chrome, & Edge. It's a combination of a Svelte bug & browser issues that is fixed in newer versions of the framework. For now I think if we add a `target="_blank"` to the grants link, it will solve the problem.

File: src/routes/grants/index.svelte [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    <div class="bx--offset-lg-2 bx--col-lg-8 bx--col-padding">
+      <p>
+        The California Energy Commission supports a variety of research projects
+        that are helping shape the development of next generation climate data
+        for California and the evolving Cal-Adapt enterprise.
+      </p>
+
+      <!-- Memo list -->
+      <div class="spacing--v-24"></div>
+      <h5>Relevant Memos</h5>
```
Comment (mukhtyar) [inline src/routes/grants/index.svelte]: Use `h2` tags for accessibility

File: src/routes/grants/index.svelte [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                  <StructuredListCell>{item.title}</StructuredListCell>
+                </StructuredListRow>
+              {/each}
+            </StructuredListBody>
+          </StructuredList>
+        </Column>
+      </Row>
+
+      <!-- Climate Research list -->
+      <h5>Ongoing Climate Research</h5>
```
Comment (mukhtyar) [inline src/routes/grants/index.svelte]: Use `h2` tags for accessibility

**Comment by melijimenez11** [comment]
@mukhtyar @elehmer I added `target="_blank"` in `commit 2c5e0d22f03cf12ca260cec5de71f58b4bc89e50`

File: src/routes/grants/index.svelte [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    const events = await this.fetch(`events.json`)
+      .then((r) => r.json())
+      .then((events) => events);
+    return {
+      posts,
+      events,
+    };
+  }
+
+  import memosList from "../../../content/grants/memos";
```
Comment (mukhtyar) [inline src/routes/grants/index.svelte]: Fetch memos and projects in the preload function.

File: src/routes/grants/index.svelte (hunk 1 above)
Comment (melijimenez11) [reply src/routes/grants/index.svelte]: Resolved in `commit fdf929093cb43423edfdc12ce18b917e75275ee4`

File: src/routes/grants/index.svelte (hunk 3 above)
Comment (melijimenez11) [reply src/routes/grants/index.svelte]: @mukhtyar I tried to recreate this following the format the homepage uses, to no avail. Do you have any more examples of how you would load these in the preload function?

File: src/routes/grants/index.svelte (hunk 2 above)
Comment (melijimenez11) [reply src/routes/grants/index.svelte]: Solved in `commit fdf929093cb43423edfdc12ce18b917e75275ee4`

File: src/routes/grants/index.svelte (hunk 3 above)
Comment (melijimenez11) [reply src/routes/grants/index.svelte]: Solved in `commit 4fa7acc07bc31c9d5715489c2a857ce5c11cd318`

File: content/grants/research.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    number: "EPC-20-006",
+    title: "2022 - May: Memorandum on the Development and Availability of Dynamically Downscaled Projections Using WRF",
+    url: "https://www.energy.ca.gov/media/7265"
+  },
+  {
+    agency: "California Energy Commission",
+    type: `Research Memo`,
+    number: "EPC-20-006",
+    title: "2022 - Nov: Memorandum on the Evaluation of Dynamically Downscaled GCMs Using WRF",
+    url: "https://caenergy.sharepoint.com/sites/EnvironmentalResearch/Agreements/PIR-17-012/4.%20Deliverables/Landing%20Page/cal-adapt.org"
```
Comment (elehmer) [inline content/grants/research.js]: This URL goes to a Sharepoint login page...

**Review by elehmer — APPROVED** [review]
Looks great! The scrolling issue is gone. Just the one URL that can't be gotten to...

File: src/routes/grants/index.svelte [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+                  >
+                </StructuredListRow>
+              {/each}
+            </StructuredListBody>
+          </StructuredList>
+        </Column>
+      </Row>
+
+      <!-- Climate Research list -->
+      <h2>Ongoing Climate Research</h2>
```
Comment (mukhtyar) [inline src/routes/grants/index.svelte:105]: maybe add same spacing as above here.

**Review by mukhtyar — APPROVED** [review]
Looks great!

File: content/grants/research.js (hunk 4 above)
Comment (melijimenez11) [reply content/grants/research.js]: Fixed in `commit 513b7d6`

File: src/routes/grants/index.svelte (hunk 5 above)
Comment (melijimenez11) [reply src/routes/grants/index.svelte:105]: Solved in commit `bf8664d`

**Review by mukhtyar — APPROVED** [review]

**Comment by mukhtyar** [comment]
@melijimenez11 merging this one in because Nancy needs it for her meeting tomorrow. We will go over deployment later this week.

**Pull request merged**